cate/ops/io.py

```diff
@@ -512,8 +512,8 @@ def write_geo_data_frame(gdf: gpd.GeoDataFrame,
 @op_input('drop_variables', data_type=VarNamesLike)
```

Comment:
That is the intended way

```suggestion
@op_input('drop_variables', data_type=VarNamesLike)
@op_input('secret', data_type=str, password=True)
@op_input('token', data_type=str, password=True)
```